### PR TITLE
release prep: standardize circleci

### PR DESCRIPTION
**Pull request by briangann**
* enable packaging and release
* setup publishing step

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,44 +53,46 @@ If you're self-hosting a Grafana installation you can follow the steps below to
...
 
-4. Make sure you have [configured your Grafana installation](https://grafana.com/docs/grafana/latest/administration/configuration/) to allow unsigned plugins
+4. When building locally, make sure you have [configured your Grafana installation](https://grafana.com/docs/grafana/latest/administration/configuration/#allow_loading_unsigned_plugins) to allow unsigned plugins.
+
 ```INI
 ...
 [plugins]
-allow_loading_unsigned_plugins=true
-...
+allow_loading_unsigned_plugins=grafana-k6cloud-datasource
```
Comment (robingustafsson) [inline README.md:80]: Ah, I see I misunderstood how this config setting worked 😅 Thanks for fixing it!

**Review by robingustafsson — APPROVED** [review]
The build/release process related changes in CircleCI is beyond me for reviewing, but besides that it LGTM!

Thanks for the help getting this plugin ready for signing and publishing 🙌

**Review by daniellee — APPROVED** [review]
@briangann @w1kman as have been on vacation, not sure if the plugin id change has been discussed as it breaks the upgrade path for existing users of the plugin. But think we could automatically migrate them in Grafana 8.2 (with a database migration in Grafana).

Otherwise LGTM and will approve this and publish the plugin.

**Comment by robingustafsson** [comment]
@daniellee Based on what we know we estimate the user base of the plugin to be small at present (I've only spoken with a handful of people/customers who've used it), so I'd say the ID change is ok. We'll communicate the breaking change in release notes and our next product update email (CC @ppcano).

**Comment by daniellee** [comment]
@robingustafsson great - we could do an automatic migration too if we feel it is needed.

**Pull request merged**